proto/cosmos/bank/v1beta1/genesis.proto

```diff
@@ -21,8 +21,11 @@ message GenesisState {
   repeated cosmos.base.v1beta1.Coin supply = 3
       [(gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins", (gogoproto.nullable) = false];
 
-  // denom_metadata defines the metadata of the differents coins.
+  // denom_metadata defines the metadata of the different coins.
   repeated Metadata denom_metadata = 4 [(gogoproto.nullable) = false];
+
+  // send_enabled defines the denoms where send is enabled or disabled.
```

Comment:
Or something like `// TOOD: Since: XXX`. We need to put something and be consistent with that. Otherwise we have more troubles to identify new things.

Reply:
Done.

I went with `// Since: cosmos-sdk 0.47` and also changed the deprecation comment to reference `cosmos-sdk 0.47`. So they're consistent and hopefully easier to find if they need to be updated due to not actually going into v0.47.